org.eclipse.wildwebdeveloper/src/org/eclipse/wildwebdeveloper/debug/chrome/ChromeRunDAPDebugDelegate.java

```diff
@@ -116,28 +114,33 @@ static File findDebugAdapter() {
 	}
 
 	static String findChromeLocation(ILaunchConfiguration configuration) {
-		String res = null;
+		String res = "chromium-browser"; //$NON-NLS-1$
 		try {
-			if (configuration.getAttribute(RUNTIME_EXECUTABLE, "no runtime executable set").equals(CHROMIUM)) {
-				res = InitializeLaunchConfigurations.which("chromium-browser");
-			} else {
-				res = InitializeLaunchConfigurations.which("google-chrome-stable");
-			}
+			res = configuration.getAttribute(RUNTIME_EXECUTABLE, "chromium-browser"); //$NON-NLS-1$
 		} catch (CoreException e) {
 			IStatus errorStatus = new Status(IStatus.ERROR, Activator.PLUGIN_ID, e.getMessage(), e);
 			Activator.getDefault().getLog().log(errorStatus);
 		}
+		if (new File(res).isAbsolute()) {
+			return res;
+		}
+		if (res == null || res.isEmpty()) {
+			res = "chromium-browser"; //$NON-NLS-1$
```

Comment:
Just to make sure I understand, inthe case that [default] is selected, chromium will be used? (I approve of this choice)